### DISCUSSION TITLE
How to install Avalonia into Vs2022 for MAC?

### DISCUSSION THREAD

**Discussion by biapar** [AvaloniaUI/Avalonia#8798] [Q&A]
How to install Avalonia into Vs2022 for MAC?
Is't support i?

**Comment by timunie** [comment]
https://docs.avaloniaui.net/docs/getting-started/ide-support

**Comment by maxkatz6** [comment]
You can build and run avalonia apps in VS MAC. Just install nuget packages. And templates can be created from the console. 
But there is no XAML intellisense and previewer support.

**Reply by timunie** [reply]
You may want to try this for preview: https://github.com/kuiperzone/AvantGarde
